ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -219,6 +219,21 @@ router bgp {{ router_bgp.as }}
 {%             endif %}
    {{ maximum_routes_cli }}
 {%         endif %}
+{%         for direction in ['in', 'out'] %}
```

Comment:
I think, we should check `if peer_group.missing_policy_af_all is arista.avd.defined` here then only it should go in the for loop.
same below